components/FilterButtons/FilterButtons.Styled.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+
+export const StyledFilterButtons = styled.button`
+  text-align: center;
+  font-size: 0.75rem;
+  padding: 8px;
+  font-weight: 400;
+  border-radius: 99rem;
```

Comment:
For a fully rounded element, 50% creates a perfect circle for square elements. 

```suggestion
border-radius: 50%; 
```

Reply:
good advice